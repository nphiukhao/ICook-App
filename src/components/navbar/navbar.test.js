import React from 'react'
import Navbar from './navbar'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`nav bar component`, () => {
    it('nav bar snapshot test', () => {
        const tree = shallow(<Navbar/>)
        expect(toJson(tree)).toMatchSnapshot()
    })
})