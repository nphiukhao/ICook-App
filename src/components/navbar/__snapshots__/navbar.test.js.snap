// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav bar component nav bar snapshot test 1`] = `
<nav
  className="navbar"
>
  <div
    className="logolink"
  >
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/"
    >
       ICook
    </Link>
  </div>
  <div
    className="navlinks"
  >
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/add"
    >
      + new recipe
    </Link>
    <Link
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/"
    >
      logout
    </Link>
  </div>
</nav>
`;
